packer/template-ubuntu-14.04.json

```diff
@@ -94,33 +104,37 @@
         [ "modifyvm", "{{.Name}}", "--nic2", "intnet" ]
       ]
       },
+
       {
         "type": "vmware-iso",
+        "vm_name": "{{user `vm_name`}}",
         "boot_command": [
           "<esc><wait>",
           "<esc><wait>",
           "<enter><wait>",
-          "/install/vmlinuz<wait>",
-          " auto<wait>",
-          " console-setup/ask_detect=false<wait>",
-          " console-setup/layoutcode=us<wait>",
-          " console-setup/modelcode=pc105<wait>",
-          " debian-installer=en_US<wait>",
-          " fb=false<wait>",
-          " initrd=/install/initrd.gz<wait>",
-          " kbd-chooser/method=us<wait>",
-          " keyboard-configuration/layout=USA<wait>",
-          " keyboard-configuration/variant=USA<wait>",
-          " locale=en_US<wait>",
-          " netcfg/get_hostname=rackhd-demo<wait>",
-          " noapic<wait>",
-          " preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed.cfg<wait>",
+          "/install/vmlinuz ",
```

Comment:
<wait> adds 1 second pause before sending any additional keys. This is useful if you have to generally wait for the UI to update before typing more.
because those lines are "command line boot command typing" in a single UI , so no need for <wait>